modules/order/templates/commerce-order--default.html.twig

```diff
@@ -0,0 +1,77 @@
+{#
+/**
+ * @file
+ * Default order template.
```

Comment:
Let's add something like "Used on the admin order page." so that the themers know which template they need.